### PR TITLE
Create initial database tables

### PR DESCRIPTION
**Pull request by bflaskrud**
Created schema and database structure as recommended by lead.

https://github.com/university-bootcamp/bootcamp-materials/blob/master/team-project/week-0/schema.rb#L1

### CODE REVIEW

File: spec/models/game_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Game, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (BryceCreamCone) [inline spec/models/game_spec.rb]: This same thing happened to me! Make sure to run the tests with bundle exec rspec and make sure they are all still passing. In this case it looks like we can either just delete this line or add a passing test.

File: spec/models/piece_spec.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Piece, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (BryceCreamCone) [inline spec/models/piece_spec.rb]: Same thing here as above

File: spec/models/game_spec.rb (hunk 1 above)
Comment (bflaskrud) [reply spec/models/game_spec.rb]: Do we have a preference if we'd like test or prefer deleting? Looks like with User/Devise we deleted? Thanks for feedback!

File: spec/models/piece_spec.rb (hunk 2 above)
Comment (bflaskrud) [reply spec/models/piece_spec.rb]: Same question as above

File: spec/models/game_spec.rb (hunk 1 above)
Comment (BryceCreamCone) [reply spec/models/game_spec.rb]: I did go ahead and delete mine. I think that would be fine to do here until we really need any testing.

Comment (bflaskrud) [reply spec/models/game_spec.rb]: Good Morning, changes should be completed. Let me know if I'm missing anything. Thanks again

File: spec/models/piece_spec.rb (hunk 2 above)
Comment (bflaskrud) [reply spec/models/piece_spec.rb]: Same comment as above

File: spec/models/game_spec.rb (hunk 1 above)
Comment (BryceCreamCone) [reply spec/models/game_spec.rb]: Good morning! I believe your branch was forked from a non-master branch. Running the current tests fail as it appears the databases were created on top of one of the unfinished angular branches. We do not want to currently merge this into the master branch

Comment (BryceCreamCone) [reply spec/models/game_spec.rb]: I totally missed this the first time, sorry. Serves me right for not actually pull the branch and running the tests. I just knew these two pending ones wouldn't work.

**Comment by BryceCreamCone** [comment]
We'll want to just revert this branch, or delete it and start a new one quick with its base being the master branch.

**Pull request merged**